test/metabase/upload_test.clj

```diff
@@ -404,14 +413,65 @@
     (testing "semicolons are removed"
       (is (nil? (re-find #";" (@#'upload/unique-table-name driver/*driver* "some text; -- DROP TABLE.csv")))))))
 
+(defn upload-example-csv!
+  "Upload a small CSV file to the given collection ID. `grant-permission?` controls whether the
+  current user is granted data permissions to the database."
+  [& {:keys [table-prefix collection-id grant-permission? uploads-enabled user-id db-id sync-synchronously? csv-file-name]
+      :or {collection-id       nil ;; root collection
+           grant-permission?   true
+           uploads-enabled     true
+           user-id             (mt/user->id :rasta)
+           db-id               (mt/id)
+           sync-synchronously? true
+           ;; Make the file-name unique so the table names don't collide
```

Comment:
The collision happens at the schema-level. Because names are created according to the second, if there are two tests that happen to run at the same instant, the names will not be unique.

Reply:
I realize my confusion - the file name *on disk* will be unique without the uuid, because this local variable here is really just the prefix. The table name just depends on the prefix, not on the filename.

I think this could be slightly clearer if you changed "file-name" to "prefix", but not a big deal.

Reply:
ok, changed file-name to prefix to avoid confusion in the future 83a10b480c7de7e03209da84c5205e0ef32c13ce

Reply:
I think you can remove the uuid here now as well. Also my OCD would prefer you didn't have whitespace in a filename 😆

Reply:
I can't remove the random UUID because of the redshift issue. I'll clean this up after https://github.com/metabase/metabase/pull/38453

Reply:
Actually now that I think of it, the chance of a collision doesn't change after https://github.com/metabase/metabase/pull/38453. If test table names are created at the same second in two concurrent test runs, the names will still collide.